src/main/java/seedu/address/model/person/Person.java

```diff
@@ -120,6 +120,59 @@ public Observable getApptsProperty() {
         return appointments;
     }
 
+    /**
+     * Returns -1 if this person appears before the other person, and
+     * returns 0 if this person has the same order as the other person, and
+     * returns 1 if this person appears after the other person.
+     *
+     * @param person The other person to compare with.
+     */
+    public int compareTo(Person person) {
+        return this.getName().compareTo(person.getName()) < 0
+                ? -1
+                : this.getName().compareTo(person.getName()) > 0
+                ? 1
+                : 0;
+    }
+
+    /**
+     * Returns the group number where this person belongs to, which is determined
+     * by its tags.
+     *
+     */
+    public Integer getGroupNumber() {
```

Comment:
Interesting use of Integer wrapper class instead of int, any particular reason why as I think it presents some overhead, although functionally the we may not be able to tell the performance difference.

Reply:
Ohh I used to call the compareTo method in Integer object, but now it seems not necessary anymore so I just changed it. Thanks!